### PR TITLE
fixes Integrations button 

### LINKED ISSUE #1633: Show All Integerations button states
### Current Behavior

1. On the web view **Show All Integrations** button has cursor as text.
2. When the button is clicked we are able to see all the integrations this is fine but to  go to previous state that is hide some integrations we need to click on **Show All Integrations** button. 
3. No spacing in mobile view between buttons and integrations grid.

### Expected Behavior

1. **Show All Integrations** button should have cursor as pointer.
2. Button text should change  like Show More Integrations  <-> Show Less Integrations. 
Show More Integration when only few integrations has been shown on the page , Show Less integration when all the integrations has been shown on the page.
4. Spacing in mobile view between buttons and integrations grid.

### Screenshots

image
image

### Environment:
 - OS: Linux Windows
 - Browser: Chrome
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by thisisharsh7**
**Description**

This PR fixes #1633 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by saurabh100ni** [comment]
Hi @thisisharsh7  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

File: _includes/partials/homePage-integrations.html [hunk 1]
```diff
@@ -67,6 +67,16 @@ <h2>
 
     document.querySelector('.show-all-button').addEventListener('click', () => {
         allDataShown = !allDataShown;
+
+        const button = document.querySelector('.toggle-text');  
+    
+        if (allDataShown) {
+            button.textContent = 'Show Less Integration'; // Change button text when allDataShown is true
```
Comment (vishalvivekm) [inline _includes/partials/homePage-integrations.html:74]:
```suggestion
            button.textContent = 'See Less Integrations'; // Change button text when allDataShown is true
```

File: _includes/partials/homePage-integrations.html [hunk 2]
```diff
@@ -67,6 +67,16 @@ <h2>
...
+        } else {
+            button.textContent = 'Show More Integrations'; // Change button text when allDataShown is false
```
Comment (vishalvivekm) [inline _includes/partials/homePage-integrations.html:76]:
```suggestion
            button.textContent = 'See All Integrations'; // Change button text when allDataShown is false
```

Comment (thisisharsh7) [reply _includes/partials/homePage-integrations.html:76]: Thanks for the correction

**Comment by sudhanshutech** [comment]
@thisisharsh7 ?

**Comment by thisisharsh7** [comment]
> @thisisharsh7 ?

Yes

**Comment by sudhanshutech** [comment]
@thisisharsh7 are we done here?

**Comment by thisisharsh7** [comment]
> @thisisharsh7 are we done here?

I had created a pr several days ago and attended two meshery meetings for explaining my pr but didn't got a chance. Please merge it if possible.

**Review by iArchitSharma — APPROVED** [review]
Awesome, thanks for fixing this, @thisisharsh7

**Pull request merged**